app/src/main/java/com/google/android/apps/youtube/app/settings/videoquality/VideoQualitySettingsActivity.java

```diff
@@ -74,7 +78,87 @@ protected void onCreate(Bundle bundle) {
+
+            // Set the margins (in pixels)
+            int margin = (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 20, getResources().getDisplayMetrics()); // for example, 10dp
+            layoutParams.setMargins(margin, layoutParams.topMargin, margin, layoutParams.bottomMargin);
+
+            // Apply the layout parameters to the SearchView
+            searchView.setLayoutParams(layoutParams);
+
+            // endregion
+
+            // region SearchView color
+
+            GradientDrawable shape = new GradientDrawable();
+
+            String currentDarkHex = DrawableColorPatch.getDarkBackgroundHexValue();
+            String currentLightHex = DrawableColorPatch.getLightBackgroundHexValue();
+
+            String currentHex = ThemeUtils.isDarkTheme() ? currentDarkHex : currentLightHex;
+            String defaultHex = ThemeUtils.isDarkTheme() ? "#1A1A1A" : "#E5E5E5";
+
+            String finalHex;
+            if (currentHex.equals(ThemeUtils.isDarkTheme() ? "#000000" : "#FFFFFF")) {
+                shape.setColor(Color.parseColor(defaultHex)); // stock black/white color
+                finalHex = defaultHex;
+            } else {
+                // custom color theme
+                String adjustedColor = ThemeUtils.isDarkTheme()
+                        ? ThemeUtils.lightenColor(currentHex, 15)
+                        : ThemeUtils.darkenColor(currentHex, 15);
+                shape.setColor(Color.parseColor(adjustedColor));
+                finalHex = adjustedColor;
+            }
+            Logger.printInfo(() -> "searchbar color: " + finalHex);
+
+            shape.setCornerRadius(30 * getResources().getDisplayMetrics().density);
+            searchView.setBackground(shape);
+
+            /* TODO:
+                in order to match the original app's search bar, we'd need to change the searchbar cursor color
+                to white (#FFFFFF) if ThemeUtils.isDarkTheme(), and black (#000000) if not.
+                Currently it's always blue.
+             */
+
+            // endregion
+
+            int leftPaddingDp = 5;
+            int topPaddingDp = 5;
+            int rightPaddingDp = 5;
+            int bottomPaddingDp = 5;
+
+            int leftPaddingPx = (int) TypedValue.applyDimension(
+                    TypedValue.COMPLEX_UNIT_DIP, leftPaddingDp, getResources().getDisplayMetrics());
+            int topPaddingPx = (int) TypedValue.applyDimension(
+                    TypedValue.COMPLEX_UNIT_DIP, topPaddingDp, getResources().getDisplayMetrics());
+            int rightPaddingPx = (int) TypedValue.applyDimension(
+                    TypedValue.COMPLEX_UNIT_DIP, rightPaddingDp, getResources().getDisplayMetrics());
+            int bottomPaddingPx = (int) TypedValue.applyDimension(
+                    TypedValue.COMPLEX_UNIT_DIP, bottomPaddingDp, getResources().getDisplayMetrics());
+
+            searchView.setPadding(leftPaddingPx, topPaddingPx, rightPaddingPx, bottomPaddingPx);
```

Comment:
If the dp of leftPadding, topPadding, rightPadding, and bottomPadding are all 5dp, there is no need to change this in integrations